spec/models/dinner_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Dinner, type: :model do
+  describe 'Validations' do
+    it 'validate presence of email'
```

Comment:
Use [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers) to test ActiveRecord validations.